sinch/domains/voice/models/callouts/requests.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchRequestBaseModel
+
+
+@dataclass
+class TextToSpeechVoiceCalloutRequest(SinchRequestBaseModel):
+    destination: dict
```

Comment:
It would be nice for DX to have a model that defines the structure of this `dict`  as it's the only required parameter: they will have to fill it for sure!

Reply:
I'm planning to use something in the middle - TypedDicts:
https://mypy.readthedocs.io/en/stable/typed_dict.html
I'll do that within DEVEXP-306 (annotation already added).